README.md

```diff
@@ -31,7 +31,7 @@ $ make
 
 (Optional) run test suite:
 ```shell
-$ make check
+$ pytest
```

Comment:
Can you invoke `pytest` via `make check` as well?

Reply:
I change back to `make check`. 👌